packages/commonwealth/server/scripts/purgeCloudflareCache.ts

```diff
@@ -0,0 +1,41 @@
+import { logger } from '@hicommonwealth/core';
+import fetch from 'node-fetch';
+import { config } from '../config';
+
+const log = logger(import.meta);
+
+async function purgeCache(zoneId?: string, apiKey?: string) {
+  if (!zoneId || !apiKey) {
+    log.warn('Missing Cloudflare env variables. Skipping cache purge');
+    return;
+  }
+
+  const url = `https://api.cloudflare.com/client/v4/zones/${zoneId}/purge_cache`;
+  const body = {
+    purge_everything: true,
+  };
+  const headers = {
+    Authorization: `Bearer ${apiKey}`,
+    'Content-Type': 'application/json',
+  };
+
+  try {
+    const response = await fetch(url, {
+      method: 'POST',
+      headers: headers,
+      body: JSON.stringify(body),
+    });
+
+    const responseData = await response.json();
+    console.log('Cache purge request successful:');
+    console.log(responseData);
+  } catch (error) {
+    console.error('Error purging cache:', error.message);
+  }
+}
+
+purgeCache(config.CLOUDFLARE.ZONE_ID, config.CLOUDFLARE.API_KEY)
+  .then(() => log.info('finished cloudflare purge script'))
+  .catch((e) => {
+    log.error('cloudflare purge script failed:', e);
+  });
```

Comment:
should add `process.exit(1)` so that it exits with a non-success code which can be useful for chaining

Reply:
I believe we have an eslint rule disallowing `process.exit(1)` -> `dispose()();` is better since it will allow Rollbar to flush logs before exiting. Also note, exiting with a non-zero status code here will fail the release and therefore cancel the deployment. If we throw here we _must_ execute this script _before_ migrations otherwise migrations could successfully execute and then deployment fails leaving DB in a state that is not compatible with the app.

Reply:
I think we should just avoid throwing as this is going into the release script. Even if we put it before the migration, if it is ever moved around it could break the release. I think it is just more safe to log instead of throw.